roles/infra/tasks/main.yml

```diff
@@ -8,6 +8,49 @@
       }
     dest: "{{ terraform_project_path }}/backend.tf"
 
+- block:
+  - name: Set a temporary fact containing terraform_backend_config
+    set_fact:
+      _terraform_backend_config: "{{ terraform_backend_config }}"
+      terraform_backend_config:
+    when:
+      - terraform_backend_config.force_path_style is defined
+      - not disable_s3_force_path_style_checks
+
+  - block:
+    - name: Warn if force_path_style is used in terraform_backend_config
```

Comment:
Shouldn't this have

```
 when:
      - terraform_backend_config.force_path_style is defined
      - not disable_s3_force_path_style_checks
```
else we're unconditionally printing a warning, even when `force_path_style` is not defined?

Reply:
The `block:` has a `when:` on L50 (https://github.com/azimuth-cloud/ansible-collection-terraform/pull/17/files#diff-23dd39183b4a9d75b3a45c56977ca19e3ef6f2c27b430120c3dbb7d15d705611R50).

So yes, you're right and it does :joy: